srtparser.h

```diff
@@ -403,34 +397,37 @@ inline void SubtitleItem::extractInfo(bool keepHTML, bool doNotIgnoreNonDialogue
             {
                 if(countP!=0)
                 {
-                    if(c != '>')
-                        c = '~';
-
+                    if(c != '>'){
+                            tag += c;
+                            c = '~';
+                    }
                     else if(c == '>')
                     {
                         c = '~';
                         countP--;
+                        _styleTagCount++;
+                        if(tag[0] == '/'){
+                            tag.erase(0,1);
```

Comment:
Additionally <font></font> and all such pairs should be counted as a singular unit.

Reply:
I'm not adding the < and > characthers. I'm ignoring them and only adding till it doesn't equal >. The problem with that approach was the / characther will get added and since i'm not adding < it's /font for example and it'll always be first.